murali/docker/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM node:10-alpine
+
+RUN mkdir -p /home/node/app/node_modules && chown -R node:node /home/node/app
```

Comment:
this will trigger an alert or a failure, because the app need to mentioned as the working directory first. then you can go for creating a working directory